### PR TITLE
User login

### LINKED ISSUE #2: User Story 2, Visitor Navigation
As a visitor
I see a navigation bar
This navigation bar includes links for the following:
- a link to return to the welcome / home page of the application ("/")
- a link to browse all items for sale ("/items")
- a link to see all merchants ("/merchants")
- a link to my shopping cart ("/cart")
- a link to log in ("/login")
- a link to the user registration page ("/register")

Next to the shopping cart link I see a count of the items in my cart

### PR DESCRIPTION
**Pull request by JoseLopez235**
PR Contains:
- It has the navbar with all its links
- It has the login for the regular user 
- All Passing test

closes #2

### CODE REVIEW

File: app/controllers/sessions_controller.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  end
+
+  def create
+    @user = User.find_by(email: params[:email])
+    if @user && @user.authenticate(params[:password])
+      session[:user_id] = @user.id
+      redirect_to '/profile'
+    else
+      redirect_to '/login'
+    end
```
Comment (KieraAllen) [inline app/controllers/sessions_controller.rb:13]: we just need to add merchant and admin to complete User Story 13

File: app/views/users/show.html.erb [hunk 2]
```diff
@@ -0,0 +1 @@
+<%= content_tag :h1, "Welcome #{@user.name}!!!" %>
```
Comment (stillsheryl) [inline app/views/users/show.html.erb:1]: Such excite!!!

**Review by stillsheryl — APPROVED** [review]
Looks good.

File: app/views/home/index.html.erb [hunk 3]
```diff
@@ -0,0 +1 @@
+<%= content_tag :h1, "Home Page" %>
```
Comment (KieraAllen) [inline app/views/home/index.html.erb:1]: love this new Rails-savvy syntax!

**Review by KieraAllen — APPROVED** [review]

File: app/views/users/show.html.erb (hunk 2 above)
Comment (c-bartell) [inline app/views/users/show.html.erb:1]: This has exactly the right amount of exclamation points!

File: config/routes.rb [hunk 4]
```diff
@@ -1,6 +1,12 @@
 Rails.application.routes.draw do
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
 
+  get "/home", to: "home#index"
+
+  get "/login", to: "sessions#new"
+
```
Comment (c-bartell) [inline config/routes.rb:7]: We could probably remove this line break down the line.

**Review by c-bartell — APPROVED** [review]

**Pull request merged**